Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesVfr.vfr

```diff
@@ -272,6 +272,33 @@ formset
          endoneof;
     endif;
 
+    suppressif ideqval FeaturesData.PowerMenuShowBatteryThresholds == 0;
+        numeric varid   = FeaturesData.BatteryConfig.StartThreshold,
+                prompt  = STRING_TOKEN(STR_BATTERY_START_THRESHOLD_PROMPT),
+                help    = STRING_TOKEN(STR_BATTERY_START_THRESHOLD_HELP),
+                flags   = RESET_REQUIRED | INTERACTIVE,
+                minimum = 0,
+                maximum = 100,
+                step    = 1,
+
+                nosubmitif prompt = STRING_TOKEN(STR_BATTERY_THRESHOLD_RANGE_ERROR),
+                    ideqid FeaturesData.BatteryConfig.StartThreshold > FeaturesData.BatteryConfig.StopThreshold
+                endif

```

Comment:
If we have those, then why do we need another check in RouteConfig function?

Also `nosubmitif` does print any warning if the start > stop? It would be confusing if the value simply don't change without any warning.

Reply:
We don't need that check in RouteConfig theoretically. `nosubmitif` will throw an error when attempting to save the values before RouteConfig is called so it should in theory be enough.

Reply:
OK, then please link the commit removing the if in RouteConfig

Reply:
Removed in 4cb4c0fcf7c349524bbe6d952178448879f1f9af